specs-go/version.go

```diff
@@ -60,50 +58,58 @@ var validSpecVersions = requiredVersionMap{
 	v080: requiresV080,
 }
 
+// ValidateVersion checks whether the specified spec version is supported.
+func ValidateVersion(version string) error {
+	if !validSpecVersions.isValidVersion(version) {
+		return fmt.Errorf("invalid version %q", version)
+	}
+	return nil
+}
+
 // MinimumRequiredVersion determines the minimum spec version for the input spec.
-func MinimumRequiredVersion(spec *cdi.Spec) (string, error) {
+func MinimumRequiredVersion(spec *Spec) (string, error) {
 	minVersion := validSpecVersions.requiredVersion(spec)
 	return minVersion.String(), nil
 }
 
-// version represents a semantic version string
-type version string
+// Version represents a semantic version string
+type Version string
```

Comment:
If we only expose `ValidateVersion` and `MinimumRequiredVersion` as the APIs here, we don't need to make this public.

Reply:
Thank you for pointing out! done.

Reply:
BTW, this can be also considered as an API break as this type was public.

Reply:
This type was not public. See https://github.com/cncf-tags/container-device-interface/blob/e66544063aa7760c4ea6330ce9e6c757f8e61df2/pkg/cdi/version.go#L70

Reply:
ah, ok. thanks for pointing out.